lib/command/init.js

```diff
@@ -168,12 +168,11 @@ module.exports = function (initPath) {
 
     const finish = async () => {
       // create steps file by default
-      if (!isTypeScript) { // no extra step file for typescript (as it doesn't match TS conventions)
-        const stepFile = `./steps_file.${extension}`;
-        fs.writeFileSync(path.join(testsPath, stepFile), defaultActor);
-        config.include.I = stepFile;
-        print(`Steps file created at ${stepFile}`);
-      }
+      // no extra step file for typescript (as it doesn't match TS conventions)
```

Comment:
Is this comment still accurate, or should it be removed?

Reply:
I'm not sure why it's there in the past. Perhaps not relevant anymore.

Reply:
Hmm, yeah, the PR/commit that introduced it doesn't seem enlightening enough to make any more sense of it

https://github.com/codeceptjs/CodeceptJS/commit/9d2ead7e7947518de9b044b3996811f1559cb14f#diff-ee8f2604bc47be2c76c3723717364e9e20858f37cf5ea2296087438abdb48dffR174

Feel free to resolve this thread as you you see fit! 😄

Reply:
it is still valid as TS doesn't like how we add extra functions into actor